Add introduction doc (#459)
oam-dev/kubevela: README.md

# KubeVela

> For developers, KubeVela is a easy-to-use tool that enables developers to describe and ship their applications with simple commands.

Reviewer: ```suggestion
For developers, KubeVela is an easy-to-use tool that enables developers to describe and ship their applications.
```

> For platform engineers, KubeVela is a fully extensible engine that allows platform builders to create something more complex on top of, with easy and Kubernetes native approach.

Reviewer: ```suggestion
For platform engineers, KubeVela is an extensible engine where platform builders can build more abstractions in a simplified and Kubernetes native approach.
```
